easyDiffractionApp/Gui/Pages/Analysis/SideBarAdvanced.qml

```diff
@@ -83,47 +126,207 @@ EaComponents.SideBarColumn {
                 }
             }
         }
-
     }
 
-    /*
     EaElements.GroupBox {
-        title: qsTr("Plot settings")
+        title: qsTr("Fitting constraints")
+        enabled: ExGlobals.Constants.proxy.experimentLoaded
+        last: true
 
-        Row {
-            spacing: EaStyle.Sizes.fontPixelSize
+        ExComponents.AnalysisConstraints {}
 
-            EaElements.CheckBox {
-                text: qsTr("Show legend")
-                checked: ExGlobals.Variables.showLegend
-                onCheckedChanged: ExGlobals.Variables.showLegend = checked
-            }
+        Column {
+            spacing: EaStyle.Sizes.fontPixelSize * 0.5
 
-            EaElements.CheckBox {
-                text: qsTr("Show measured")
-                checked: ExGlobals.Constants.proxy.showMeasuredSeries
-                onCheckedChanged: ExGlobals.Constants.proxy.showMeasuredSeries = checked
-            }
+            // Type 1 START
+            Column {
+                spacing: EaStyle.Sizes.fontPixelSize * -0.5
 
-            EaElements.CheckBox {
-                text: qsTr("Show difference")
-                checked: ExGlobals.Constants.proxy.showDifferenceChart
-                onCheckedChanged: ExGlobals.Constants.proxy.showDifferenceChart = checked
+                EaElements.Label {
+                    enabled: false
+                    text: qsTr("Type 1")
```

Comment:
This should probably be more correctly named, maybe `Single parameter constraint` and `Two parameter constraint` for Type 2?